math/dlib/Portfile

```diff
@@ -33,3 +38,50 @@ compiler.blacklist-append {clang < 800}
 
 configure.cxxflags-append -fvisibility=hidden -fvisibility-inlines-hidden
 cmake.build_type    Release
+
+# create Python subports
+set python_versions {37 38}
+foreach v ${python_versions} {
+    subport py${v}-${github.project} {
+        PortGroup   python 1.0
+        python.default_version  ${v}
+    }
+}
+
+# Python bindings for supported Python versions
+if {[string match "py*" ${subport}]} {
+    github.setup    ${github.author} ${github.project} ${github.version} ${github.tag_prefix}
```

Comment:
It looks like you're just setting all the variables to their existing values again. What does this accomplish?

Reply:
Fixed thanks.